src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -103,6 +104,15 @@ public void Check(TrackedDownload trackedDownload)
                 if (seriesMatchType == SeriesMatchType.Id)
                 {
                     trackedDownload.Warn("Found matching series via grab history, but release was matched to series by ID. Automatic import is not possible.");
+                    trackedDownload.State = TrackedDownloadState.ImportPendingNotified;
```

Comment:
I don’t think this should be a separate state, if it needs to be tracked here then a separate property would be more appropriate.